cmake/modules/hunter_autotools_project.cmake

```diff
@@ -94,7 +99,17 @@ function(hunter_autotools_project target_name)
   hunter_test_string_not_empty("${PARAM_BUILD_DIR}")
   hunter_test_string_not_empty("${PARAM_GLOBAL_INSTALL_DIR}")
   hunter_test_string_not_empty("${PARAM_INSTALL_DIR}")
+  hunter_test_string_not_empty("${PARAM_PACKAGE_CONFIGURATION_TYPES}")
 
+  list(LENGTH PARAM_PACKAGE_CONFIGURATION_TYPES _len)
```

Comment:
We are inside the function so it's okay to use "normal" variable names without "_" at start (see [Naming](http://0.readthedocs.io/en/latest/cmake.html#naming)).